Sources/SwiftSyntaxParser/SyntaxParser.swift

```diff
@@ -286,6 +287,3 @@ extension Diagnostic {
   }
 }
 
-fileprivate func verifyCNodeLayoutHash() -> Bool {
-  return cNodeLayoutHash() == SwiftSyntax.cNodeLayoutHash()
```

Comment:
Now that `SwiftSyntax` doesn't know anything about `CNodes`, so this is not needed anymore.